_posts/2020-07-08-pull-requests-require-context.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: epic
+title: "Pull Requests Need Context"
```

Comment:
_Every pull request should tell a story_
_When it comes to Pull Requests, show, don't tell_
_This one weird trick will make code review go better and faster_

Reply:
_This is the type of Pull Requests I want to review from now on_

Reply:
Thanks for the suggestions! They helped me arrive at _Improve Pull Requests By Including Valuable Context_.